### PR TITLE
Add aliases to allow autowiring

### PR DESCRIPTION
**Pull request by richardhj**
## Description

I want to use autowiring on an app basis and these aliases are necessary

## Checklist
- [x] Read and understood the [CONTRIBUTING guidelines](CONTRIBUTING.md)
- [x] Created tests, if possible
- [x] All tests passing
- [x] Extended the README / documentation, if necessary
- [x] Added myself to the `@authors` in touched PHP files
- [x] Checked the changes with phpcq and introduced no new issues

### CODE REVIEW

**Comment by zonky2** [comment]
@discordier it´s possible to work with your autofixer?

File: src/CoreBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 services:
+    MetaModels\IFactory: '@metamodels.factory'
+    MetaModels\Filter\Setting\FilterSettingFactory: '@metamodels.filter_setting_factory'
```
Comment (discordier) [inline src/CoreBundle/Resources/config/services.yml]: Shouldn't this be:
```diff
- MetaModels\Filter\Setting\FilterSettingFactory: '@metamodels.filter_setting_factory'
+ MetaModels\Filter\Setting\IFilterSettingFactory: '@metamodels.filter_setting_factory'
- MetaModels\Render\Setting\RenderSettingFactory: '@metamodels.render_setting_factory'
+ MetaModels\Render\Setting\IRenderSettingFactory: '@metamodels.render_setting_factory'
```
as these are also interface

Comment (richardhj) [reply src/CoreBundle/Resources/config/services.yml]: What more interfaces? `IItem`and `IItems` should not be injected and `IMetaModelsServiceContainer` is deprecated.

**Comment by zonky2** [comment]
@richardhj can you push this at gitlab? base ist `feature/2.2.0`

**Pull request closed without merging**